<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>treesClass</class>
 <widget class="QMainWindow" name="treesClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>trees</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="CalAvgButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>计算平均查找长度</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>371</width>
      <height>111</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>在此输入数列，数之间用空格分隔</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CreateBSTButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>生成BST树</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CreateAVLButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>160</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>生成AVL树</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowTreeButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>查看树状存储</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowArrayButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>240</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>查看数组存储</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="InsertNewEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>300</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="InsertNewButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>300</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>插入新元素</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="DeleteEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>查找并删除元素</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ReadFileButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>从文件读入</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LNRbutton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>240</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>中序遍历</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>668</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
